### DISCUSSION TITLE
No-Number Math Mode

### DISCUSSION THREAD

**Discussion by isaacew** [typst/typst#3971] [Ideas]
I thought it would be nice to have an option for a no-number block math mode. The syntax could follow what LaTeX does where 

```
*$ f(x) $* 
```

Would be a non numbered block equation. 

```
$ g(x) $ 
```

Would be a numbered block equation. 

```
$h(x)$ 
```

would remain as an in-line equation. 

I know it looks like it conflicts a bit with the bold syntax and understand if developers don't like it. But, I think the idea has merit as the present way to do no-numbered equations in the midst of numbered ones is to use something like this 

```
#math.equation($ f(x) $,block: true, numbering: none)
```

Which is harder to read. But works nonetheless.

**Comment by Enter-tainer** [comment]
you may want to wrap the code with backticks. currently it's bearly readable

**Comment by isaacew** [comment]
> you may want to wrap the code with backticks. currently it's bearly readable

Thanks. I fixed it.

**Comment by laurmaedje** [comment]
The current situation is not ideal, but I'm not a big fan of adding more special syntax, especially since there are also various other things one might want to change about an equation.

Related: https://github.com/typst/typst/issues/1921
